cmd/top-net-spinner.go

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2015-2022 MinIO, Inc.
+//
+// This file is part of MinIO Object Storage stack
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package cmd
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/charmbracelet/bubbles/spinner"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/dustin/go-humanize"
+	"github.com/minio/madmin-go/v3"
+	"github.com/olekukonko/tablewriter"
+)
+
+type topNetUI struct {
+	spinner  spinner.Model
+	quitting bool
+
+	sortAsc   bool
+	count     int
+	endPoints []string
+
+	prevTopMap map[string]*madmin.NetMetrics
+	currTopMap map[string]*madmin.NetMetrics
```

Comment:
```suggestion
	currTopMap map[string]madmin.NetMetrics
```
Having a pointer provides no value for you, since a nil value doesn't make sense.